_toc.yml

```
@@ -18,5 +19,6 @@ parts:
   - caption: DevOps Lifecycle
     chapters:
     - file: docs/source/deploy-model
+    - file: docs/source/model-deployment/flask-application
```

```suggestion
    - file: docs/source/deploy-model
      sections:
        - file: docs/source/model-deployment/flask-application
```
What if we put the deploy-as-a-flask-application page as a subsection of deploy-your-models? To explicitly show that it's just _one_ of the methods of  model deployment.

---

sounds good! this gives space to add more methods, if wanted :) i like it